solutions/tests/test_steps_to_floor.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+from ..steps_to_floor import steps_to_floor
+
+
+class TestStepsToFloor(unittest.TestCase):
+    def test_zero_floor(self):
+        self.assertEqual(steps_to_floor(0), 0)
+
+    def test_positive_floor(self):
+        self.assertEqual(steps_to_floor(1), 10)
+        self.assertEqual(steps_to_floor(5), 50)
+
+    def test_negative_floor(self):
+        self.assertEqual(steps_to_floor(-1), 10)
+        self.assertEqual(steps_to_floor(-3), 30)
+
+    def test_invalid_input(self):
```

Comment:
these are defensive test cases i'd recommend adding a comment to show thatm it'll help alot.